lte/gateway/c/oai/tasks/nas/emm/Attach.c

```diff
@@ -966,11 +966,6 @@ static int _emm_attach_release(emm_context_t* emm_context) {
         LOG_NAS_EMM,
         "EMM-PROC  - Release UE context data (ue_id=" MME_UE_S1AP_ID_FMT ")\n",
         ue_id);
-
```

Comment:
Where is this cleanup happening if it is not happening here?

Reply:
As part of Security Mode Reject, mme was clearing emm context and initiating UE context release request  towards eNB to release s1ap signaling contexts. On reception of UE context release complete, it calls mme_remove_ue_context(), in which it removes UE context from hashlist, imsi_mme_ue_id_htbl based on ue_context_p->emm_context._imsi64; Since the content of emm context are cleared while handling Security Mode Reject,  so mme was not able to remove UE context from hashlist, imsi_mme_ue_id_htbl.

If emm_contexts get cleared on reception of UE context release complete, need not clear emm context here